backend/src/main/java/com/yigongil/backend/domain/member/Member.java

```diff
@@ -52,7 +52,7 @@ public Member(
         this.githubId = githubId;
         this.nickname = new Nickname(nickname);
         this.profileImageUrl = profileImageUrl;
-        this.tier = tier;
+        this.tier = tier != null ? tier : 1;
```

Comment:
p4: 사소한 제안인데, 조건문을 긍정형으로 넣으면 어떨까요??
저는 처음 봤을 때 'tier가 null이 아닌 게 아니면 1을 넣어준다' 라는 의미로 해석이 되더라고요.
'tier가 null이면 1을 넣어준다'라는 게 더 직관적으로 보일 것 같아요.

```suggestion
        this.tier = tier == null ? 1 : tier;
```

Reply:
좋네요 수정했습니다